doc/cli/operator-sdk_cleanup.md

```diff
@@ -21,6 +21,7 @@ operator-sdk cleanup [flags]
       --install-mode string       [olm only] InstallMode to create OperatorGroup with. Format: InstallModeType=[ns1,ns2[, ...]]
       --include strings           [olm only] Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by run/cleanup
       --timeout duration          [olm only] Time to wait for the command to complete before failing (default 2m0s)
+      --olm-namespace string      [olm only] Namespace where OLM is installed (default "olm")
```

Comment:
I think we need to make clear that it is the namespace where the OLM and the operator OLM files will be installed.. WDYT?

Reply:
I think the `--olm-namespace` flag's description as  "Namespace where OLM is installed". 

For `operator-sdk olm status` subcommand  this makes correct sense because its checking whether OLM is installed in that namespace.

For `operartor-sdk run & cleanup --olm` subcommands , I agree its doing two things 1 - checks if OLM is installed and 2 - creates the following resources in the olm namespace.

```
registry server
 configmap - operator-registry-bundles
 deployment - operator-registry-server
 service - operator-registry-server
operator resources
 CatalogSource
 Subscription
 OperatorGroup
```

I don't know what's the reason behind installing these resources in this `olm` namespace, because the from docs I generally these resources are generally installed to `marketplace` namespace ( `openshift-marketplace` on ocp clusters). I think the intention here is to install in a global catalog namespace https://github.com/operator-framework/community-operators/issues/589#issuecomment-520888065. 

So I think we should leave this flag's purpose to point to the namespace where olm is installed. May be change the namespace where this resources created ( may be marketplace) - need some expert eyes on this. Or have a flag (ex: --catalog-namespace) to choose where this resources are created

Reply:
Fair enough 👍